### ISSUE TITLE
Potato list all throws ModuleNotFoundError error

### ISSUE THREAD

**Issue by vitthal-bhandari** [davidjurgens/potato#40]
I installed the package using pip 

`pip install potato-annotation`

Then I ran the command below

`potato list all`

I received the error below:

```
D:\potato>potato list all
Traceback (most recent call last):
  File "c:\python38\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\python38\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Python38\Scripts\potato.exe\__main__.py", line 4, in <module>
  File "c:\python38\lib\site-packages\potato\cli.py", line 2, in <module>
    from potato.flask_server import main
  File "c:\python38\lib\site-packages\potato\flask_server.py", line 19, in <module>
    import simpledorff
ModuleNotFoundError: No module named 'simpledorff'
```

OS: **Windows**
Python version: **3.8.3**
Pip version:  **21.1.3**

**Comment by Jiaxin-Pei** [comment]
Hi @vitthal-bhandari,

You could simply `pip install simpledorff` to resolve this issue. We will include the package dependency in the next version of potato pypi package.

**Comment by vitthal-bhandari** [comment]
Hi @Jiaxin-Pei 

I tried your solution but got a second dependency error:

```
D:\potato>pip install simpledorff
Collecting simpledorff
  Downloading simpledorff-0.0.2-py3-none-any.whl (5.6 kB)
Installing collected packages: simpledorff
Successfully installed simpledorff-0.0.2
WARNING: You are using pip version 21.1.3; however, version 23.0.1 is available.
You should consider upgrading via the 'c:\python38\python.exe -m pip install --upgrade pip' command.

D:\potato>potato list all
Traceback (most recent call last):
  File "c:\python38\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\python38\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Python38\Scripts\potato.exe\__main__.py", line 4, in <module>
  File "c:\python38\lib\site-packages\potato\cli.py", line 2, in <module>
    from potato.flask_server import main
  File "c:\python38\lib\site-packages\potato\flask_server.py", line 24, in <module>
    from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'
```

**Comment by vitthal-bhandari** [comment]
Installing bs4 manually resolves the issue.

```
D:\potato>pip install bs4
Collecting bs4
  Downloading bs4-0.0.1.tar.gz (1.1 kB)
Collecting beautifulsoup4
  Downloading beautifulsoup4-4.12.0-py3-none-any.whl (132 kB)
     |████████████████████████████████| 132 kB 2.2 MB/s
Collecting soupsieve>1.2
  Downloading soupsieve-2.4-py3-none-any.whl (37 kB)
Building wheels for collected packages: bs4
  Building wheel for bs4 (setup.py) ... done
  Created wheel for bs4: filename=bs4-0.0.1-py3-none-any.whl size=1273 sha256=e3328957038f4ebaab6e386db2884fb10fe94c1a5b286abf0ca3b8e708cf52f2
  Stored in directory: c:\users\vitth\appdata\local\pip\cache\wheels\75\78\21\68b124549c9bdc94f822c02fb9aa3578a669843f9767776bca
Successfully built bs4
Installing collected packages: soupsieve, beautifulsoup4, bs4
Successfully installed beautifulsoup4-4.12.0 bs4-0.0.1 soupsieve-2.4
WARNING: You are using pip version 21.1.3; however, version 23.0.1 is available.
You should consider upgrading via the 'c:\python38\python.exe -m pip install --upgrade pip' command.

D:\potato>potato list all
all the available example projects that you can directly fetch
simple_schema_examples
dialogue_analysis
empathy
gif_reply
immigration_framing
match_finding
match_finding_with_prestudy
sentiment_analysis
summarization_evaluation
textual_uncertainty
question_answering
```

**Comment by Jiaxin-Pei** [comment]
fixed with potato-annotation 1.2.0.10, you could run `pip install potato-annotation --upgrade` to install the latest version.
